ui-particles-angular/projects/ui-policy-studio-angular/src/lib/models/json.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export type Json = string | number | boolean | null | Json[] | { [key: string]: Json };
```

Comment:
is this type exported only for this module or can it create problems in the whole app ?

Reply:
I don't think so. I don't really know what to do with these `utils` types 🤷‍♂️